fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -41,7 +41,6 @@ statement
     | createExternalCatalogStatement                                                        #createCatalog
     | dropExternalCatalogStatement                                                          #dropCatalog
     | showCatalogStatement                                                                  #showCatalogs
-    | showDbFromCatalogStatement                                                            #showDbFromCatalog
```

Comment:
why not delete showDbFromCatalogStatement?

Reply:
> why not delete showDbFromCatalogStatement?

need to wait for https://github.com/StarRocks/starrocks/pull/6045 got merged